js/API/API.js

```diff
@@ -227,16 +227,11 @@ data - The data to download.
 */
 
 function downloadFile(fileName, data) {
-    var a = document.createElement("a");
-    document.body.appendChild(a);
-	
-	var blob = new Blob([data], {type: "octet/stream"}),
-	    url = window.URL.createObjectURL(blob);
-	a.href = url;
-	a.download = fileName;
-	a.click();
-	window.URL.revokeObjectURL(url);
-	a.remove();
+	var downloadlink = document.body.appendChild(document.createElement("a"));
+	downloadlink.download = fileName;
+	downloadlink.href = "data:text/plain;base64," + btoa(unescape(encodeURIComponent(data)));
```

Comment:
I think there are some issues with size constraints on data uri's. Maybe this is no longer an issue with modern browsers but please look into it.